### PR TITLE
add facebook sdk support

### PR DESCRIPTION
**Pull request by wuzhiming**
add facebook sdk support

now facebook live stream is available
相关issue:
https://github.com/cocos-creator/fireball/issues/7642

### CODE REVIEW

File: templates/js-template-default/frameworks/runtime-src/proj.android-studio/gradle.properties [hunk 1]
```diff
@@ -40,3 +40,6 @@ RELEASE_STORE_FILE=
 RELEASE_STORE_PASSWORD=
 RELEASE_KEY_ALIAS=
 RELEASE_KEY_PASSWORD=
+
+#facebook sdk
+FACEBOOK_APP_ID=
```
Comment (drelaptop) [inline templates/js-template-default/frameworks/runtime-src/proj.android-studio/gradle.properties]: another space line is better, both __default__ and __link__ template

**Comment by drelaptop** [comment]
please check and merge @minggo

**Comment by wuzhiming** [comment]
@minggo please check and merge

**Comment by jareguo** [comment]
只改了 EOF 的空行，我直接合并啦

**Pull request merged**